### DISCUSSION TITLE
Pull request checks don't seem to get executed.

### DISCUSSION THREAD

**Discussion by hmf** [almond-sh/almond#921] [Q&A] [answered]
I have [this PR](https://github.com/almond-sh/almond/pull/918) that has Scala 3.1.1  compiling, but the checks never seem to start. Anything I can do on my end?

TIA

**Comment by alexarchambault** [comment, marked as answer]
I think once https://github.com/almond-sh/almond/pull/918 is merged, the CI of your PRs should start automatically (GitHub requires manual validation for new contributors…)
